src/Domain/Creators/SphereTimeDependentMaps.cpp

```diff
@@ -105,24 +116,63 @@ TimeDependentMapOptions::create_functions_of_time(
                                  {0.0}}},
       expiration_times.at(size_name));
 
+  // ExpansionMap FunctionOfTime
+  // Note: Missing expiration times in SettleToConstant
```

Comment:
You can remove the note. This is expected.

Reply:
Removed